src/persistence.ts

```diff
@@ -48,22 +48,22 @@ export interface ConditionalDeleteResourceRequest {
 
 export interface InitiateExportRequest {
     requesterUserId: string;
-    requestGranularity: ExportRequestGranularity;
+    requestGranularity: ExportType;

```

Comment:
I think that we should also rename the key to `exportType`

Reply:
fixed